docs/eos-learn/accounts-and-permissions.md

```diff
@@ -10,7 +10,7 @@ An **account** on EOSIO is a legible name for humans that get stored on the bloc
 
 The account name must comply with certain requirements: characters from `a` to `z` in lower case, numbers from `1` to `5` and have 12 characters length. E. g. `eosioaccount`.
 
-Read more on [EOSIO accounts](https://developers.eos.io/welcome/latest/protocol/accounts_and_permissions#2-accounts).
+Read more on [EOSIO accounts](https://eos-account-creator.com/choose/).
```

Comment:
this seems like an EOS mainnet tool , is there a more generic EOSIO resource instead?

Reply:
I'll check that and update it!

Reply:
What do you think about this link instead? [new link](https://developers.eos.io/manuals/eosjs/v21.0/how-to-guides/how-to-create-an-account)

Reply:
the link looks good , maybe we can change the name of the link text from `Read more on [EOSIO accounts]` to `Read more on [EOSIO account creation]`